src/configure-nilrt-snac

```diff
@@ -188,6 +188,30 @@ EOF
 	rmmod cfg80211 mac80211 >/dev/null 2>&1
 }
 
+configure_tmux() {
+	log INFO Install tmux
+	opkg update >/dev/null && opkg install tmux
+
+	log INFO Create timeout configuration for tmux and source it
+	mkdir -p /usr/share/nilrt-snac
+	cat <<EOF >/usr/share/nilrt-snac/tmux-snac.conf
+# NILRT SNAC configuration tmux-snac.conf. Do not hand-edit.
+
+set -g lock-after-time 900
+EOF
+	echo "source-file /usr/share/nilrt-snac/tmux-snac.conf" >>/etc/tmux.conf
```

Comment:
Thanks for making this change, I'll just note that we could get rid of this line if you prefer my [design suggestion](https://github.com/ni/meta-nilrt/pull/718#discussion_r1722189535) over in meta-nilrt.